### DISCUSSION TITLE
Can't add any blog

### DISCUSSION THREAD

**Discussion by onoczarkin** [TumblThreeApp/TumblThree#361] [Q&A (Question & Answer)]
**Describe the bug**
I just downloaded the software but I can't add any blog. For example when I type "https://www.tumblr.com/blog/jdmcarguy" in the left textfield the "Add Blog" button is just greyed out and when I click it nothing happens. Is this a bug or am i doing something wrong?

**To Reproduce**
1. Type "https://www.tumblr.com/blog/view/jdmcarguy" in the left textfield

**Expected behavior**
When I click on "Add Blog" the blog should be added but instead nothing happens.

**Desktop (please complete the following information):**
 - TumblThree version: 1.0.12.1
 - OS: Windows 10 Home

**Comment by thomas694** [comment]
You can add the blog with "https://jdmcarguy.tumblr.com".

**Comment by onoczarkin** [comment]
@thomas694 Thanks, the adding of a blog works now. But I can't seem to download a blog. I only have to add the blog to the queue and click "Scan" to download, don't I?

**Comment by thomas694** [comment]
Set/check the [download location](https://github.com/TumblThreeApp/TumblThree/wiki/Getting-Started), add the blog to the [download queue](https://github.com/TumblThreeApp/TumblThree/wiki/How-to-use-the-Application) and press Crawl. It should work.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
